packages/@react-spectrum/s2/stories/Tabs.stories.tsx

```diff
@@ -58,10 +58,10 @@ export const Example = (args: any) => (
 );
 
 export const Disabled = (args: any) => (
-  <div className={style({width: 700, height: 256, resize: 'horizontal', overflow: 'hidden', padding: 8})}>
+  <div className={style({width: 700, maxWidth: '[calc(100vw - 60px)]', height: 256, resize: 'horizontal', overflow: 'hidden', padding: 8})}>
```

Comment:
Do we need this story since we have isDisabled control and other stories with a single tab disabled?

Reply:
this disables individual keys as opposed to the whole control